theano/gof/cmodule.py

```diff
@@ -377,7 +377,7 @@ def is_same_entry(entry_1, entry_2):
 
 def get_module_hash(src_code, key):
     """
-    Return an MD5 hash that uniquely identifies a module.
+    Return a SHA256 (for FIPS compatibility) hash that uniquely identifies a module.
```

Comment:
We only care about it because it allow to use Theano on computer that are configured to be FIPS compatible from what I understand.

I'm ok to not reference it as we don't make any garanty about it in Theano.